platform/ios/src/MGLMapView.mm

```diff
@@ -2571,6 +2571,11 @@ - (void)setZoomLevel:(double)zoomLevel animated:(BOOL)animated
                       MGLDurationFromTimeInterval(duration));
 }
 
++ (NS_SET_OF(NSString *) *)keyPathsForValuesAffectingMinimumZoomLevel
+{
+    return [NSSet setWithObject:@"camera"];
```

Comment:
The `camera` key path doesn’t affect the minimum and maximum zoom level. These properties should already be hooked up sufficiently; they just need to be made inspectable.